### PR TITLE
LPS-121042 Create test to check if sector is highlighted on mouse over and update snapshot

### PR DESCRIPTION
**Pull request by alinedoleron**
Related Issue: https://issues.liferay.com/browse/LPS-121042

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @alinedoleron, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/chart/pie/PieChart.es.js [hunk 1]
```diff
@@ -64,4 +67,16 @@ describe('PieChart', () => {
 			container.querySelector('.recharts-pie-sector > path')
 		).toBeTruthy();
 	});
+
+	it('highlight a sector when mouse is over it', () => {
+		const {container} = render(<PieChart {...props} />);
+
+		const sector = container.querySelector('.recharts-pie-sector');
+		fireEvent.mouseOver(sector);
+		expect(
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-report-web/test/js/components/chart/pie/PieChart.es.js:76]: @alinedoleron, please add line breaks. Thanks.

**Comment by natocesarrego** [comment]
Great job, @alinedoleron! 😄

**Comment by alinedoleron** [comment]
new pr https://github.com/natocesarrego/liferay-portal/pull/864

**Pull request closed without merging**